Makefile

```diff
@@ -590,6 +602,23 @@ else
 endif
 endef
 
+# first argument: a configuration name from the union of config_list and
+# config_name, used to look up the CFLAGS to use during compilation.
+# second argument: the C99 addon file suffix being considered.
+# third argument: the name of the addon being considered.
+define make-c99-addon-kers-rule
+$(BASE_OBJ_ADDON_PATH)/$(3)/$(KERNELS_DIR)/%.o: $(ADDON_PATH)/$(3)/$(KERNELS_DIR)/%.$(2) $(BLIS_H_FLAT) $(ADDON_H99_FILES) $(MAKE_DEFS_MK_PATHS)
+ifeq ($(ENABLE_VERBOSE),yes)
+	$(CC) $(call get-addon-kernel-c99flags-for,$(1)) -c $$< -o $$@
```

Comment:
This rule is resulting in compilation errors since the header file paths are not part of call get-addon-kernel-c99flags-for. Adding get-addon-c99flags-for to this rule fixes the issue, but not sure if it is the right way to do it,

Reply:
You are right, Mithun! Thank you for identifying this issue and its fix. (It worked for me the first time only because I was using an dummy `.c` file in my addon `kernels` directory, and the file was empty, thus no need for the addon header paths.)

Please try my follow-up commit.

Reply:
I have tried the latest changes and it works perfectly. Thanks for the quick response.

Reply:
That's great news, Mithun. Thank you for your feedback!